src/rgw/rgw_sal_motr.cc

```diff
@@ -2645,7 +2645,7 @@ int MotrObject::open_mobj(const DoutPrefixProvider *dpp)
   struct m0_op *op = nullptr;
   mobj->ob_attr.oa_layout_id = meta.layout_id;
   mobj->ob_attr.oa_pver      = meta.pver;
-  mobj->ob_entity.en_flags  |= M0_ENF_META;
+  mobj->ob_entity.en_flags  |= (M0_ENF_META | M0_ENF_GEN_DI);
```

Comment:
Is this flag needs to be set during create, open and delete operation as well? and why?

Reply:
Its good to keep the flag setting consistent so added this, may not be used during create, open and delete.

Reply:
see the comment in Motr code here: https://github.com/Seagate/cortx-motr/blob/7741693b1663c0aa2e9d48a58f64abddc0cadd95/motr/client.h#L633, CREATE op also expects M0_ENF_META flag, so I think it is good to set it for CREATE op.